subsys/mpsl/fem/common/mpsl_fem_utils.c

```diff
@@ -5,13 +5,23 @@
  */
 
 #include <mpsl_fem_utils.h>
-
-#if defined(CONFIG_MPSL_FEM_PIN_FORWARDER)
-
 #include <stdbool.h>
 #include <string.h>
 #include <zephyr/devicetree.h>
 #include <zephyr/sys/__assert.h>
+#include <hal/nrf_gpio.h>
+
+#if !defined(CONFIG_MPSL_FEM_PIN_FORWARDER)
+
+void mpsl_fem_extended_pin_to_mpsl_fem_pin(uint8_t pin_num, mpsl_fem_pin_t *p_fem_pin)
+{
+	p_fem_pin->port_pin = pin_num;
+    // The first call overwrites pin_num, the second p_fem_pin->port_pin
+	p_fem_pin->port_no  = nrf_gpio_pin_port_number_extract(&pin_num);
```

Comment:
Twister build is failing because `uint8_t *` is used in these calls instead of expected `uint32_t *`. Please align to fix the build